uber/templates/budget/index.html

```diff
@@ -1,4 +1,5 @@
-{% extends "base-admin.html" %}
+{% set admin_area=True %}
```

Comment:
yea the eventual goal would be to kill base-admin.html and all pages inherit from base.html

However, for this PR originally I wanted to change as little as possible, so, left base-admin.html in as an option.